benchmark.go

```diff
@@ -187,13 +206,77 @@ func runBenchmark(out output, logger *zap.Logger, allOpts Options, resolved reso
 		zap.Int("totalRequests", overall.totalRequests),
 		zap.Time("startTime", start),
 	)
-
+	// Print out errors
```

Comment:
just a note here to think about. we are printing errors in plaintext regardlesss of of the output format. That means we may have plaintext mixed with json output in case of error, and make the final output not parsable. That's not critical though, given this is just the error case. maybe leave a TODO here to consider move error into json field as well.